### PR TITLE
Add inline styles section to Styling

### PR DESCRIPTION
**Pull request by Jutanium**
Discord thread link: https://discordapp.com/channels/830184174198718474/1075913546883407972

#### What kind of changes does this PR include?

- New or updated content

#### Description

- Replaces #2774
- Adds an inline styles section that shows an example of style objects

### CODE REVIEW

File: src/content/docs/en/guides/styling.mdx [hunk 1]
```diff
@@ -148,6 +148,15 @@ This pattern lets you style child components directly. Astro will pass the paren
 Because the `class` prop includes the child in its parent’s scope, it is possible for styles to cascade from parent to child. To avoid this having unintended side effects, ensure you use unique class names in the child component.
 :::
 
+### Inline styles
+
+You can style HTML elements inline using the `style` attribute. This can be a CSS string or an object of CSS properties:
+
+```astro title="src/pages/index.astro"
+//These are equivalent:
```
Comment (Yan-Thomas) [inline src/content/docs/en/guides/styling.mdx:156]: Super nit: spacing for consistency's sake!
```suggestion
// These are equivalent:
```

**Review by Yan-Thomas — COMMENTED** [review]
LGTM, thanks for working on this @WintrCodes 🙌. I just have a super small nit but after it, it's ready to merge!

File: src/content/docs/en/guides/styling.mdx [hunk 2]
```diff
@@ -148,6 +148,15 @@ This pattern lets you style child components directly. Astro will pass the paren
...
 :::
 
+### Inline styles
+
+You can style HTML elements inline using the `style` attribute. This can be a CSS string or an object of CSS properties:
+
+```astro title="src/pages/index.astro"
+// These are equivalent:
+<p style={{ color: "brown", textDecoration: "underline" }}>My text</p>
+<p style="color: brown; text-decoration: underline">My text</p>
```
Comment (sarah11918) [inline src/content/docs/en/guides/styling.mdx:158]:
```suggestion
<p style="color: brown; text-decoration: underline;">My text</p>
```
I have almost always seen a semi-colon used, even in the final property. I think I might prefer to show this. Thoughts?

**Review by sarah11918 — APPROVED** [review]
Ready for you to merge, Dan! One question below!

**Pull request merged**